pkg/local-storage/member/csi/utils.go

```diff
@@ -66,85 +63,24 @@ func pathExists(path string) (bool, error) {
 }
 
 func getVolumeMetrics(mntPoint string) (*VolumeMetrics, error) {
-	headers := []string{
-		"Inodes",
-		"IFree",
-		"IUsed",
-		"1B-blocks",
-		"Avail",
-		"Used",
-	}
-	dfFlags := []string{
-		"--sync",
-		"--block-size=1",
-		"--output=itotal,iavail,iused,size,avail,used",
-		mntPoint,
-	}
-	dfPath := "df"
-
-	res := nsexecutor.New().RunCommand(exechelper.ExecParams{
-		CmdName: dfPath,
-		CmdArgs: dfFlags,
-	})
+	var stats syscall.Statfs_t
 
-	if res.ExitCode != 0 {
-		return nil, res.Error
-	}
-	if res.OutBuf == nil {
-		return nil, fmt.Errorf("no output")
-	}
+	syscall.Sync()
 
-	for _, line := range strings.Split(res.OutBuf.String(), "\n") {
-		line = strings.TrimSpace(line)
-		// Skip for empty line or before header found.
-		if len(line) == 0 {
-			continue
-		}
+	err := syscall.Statfs(mntPoint, &stats)
```

Comment:
This 'syscall' will be executed inside the local-storage container, and will not get the expected result. Actually, the command should be executed on the OS which is outside of the container, that's why it is executed by nsexecutor.
In addition, did you test your change?